aws/resource_aws_glue_job.go

```diff
@@ -44,6 +44,10 @@ func resourceAwsGlueJob() *schema.Resource {
 							Type:     schema.TypeString,
 							Required: true,
 						},
+						"python_version": {
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
Let’s add a validation check for this attribute.  `ValidateFunc: validation.IntInSlice([]int{2,3}),`